docs/whats-new/2020-11.md

```diff
@@ -0,0 +1,160 @@
+---
+title: ".NET docs: What's new for November 1, 2020 - November 30, 2020"
+description: "What's new in the .NET docs for November 1, 2020 - November 30, 2020."
+ms.date: 12/02/2020
+---
+
+# .NET docs: What's new for November 1, 2020 - November 30, 2020
+
+Welcome to what's new in the .NET docs from November 1, 2020 through November 30, 2020. This article lists some of the major changes to docs during this period.
+
+## .NET
```

Comment:
Why do we differentiate .NET and .NET fundamentals?

Reply:
We probably shouldn't.  What are your thoughts @tdykstra ?

Reply:
I agree, we shouldn't differentiate. The division is based on standard vs. core TOC groupings, which are now combined in fundamentals.

Reply:
Done. I combined them by hand for this month. We'll make a code change to fix it for next month.